frontend/src/metabase/components/Modal/WindowModal.tsx

```diff
@@ -73,10 +73,7 @@ export class WindowModal extends Component<WindowModalProps> {
         closeOnClickOutside={this.props.closeOnClickOutside}
       >
         <FocusTrap active={this.props.trapFocus}>
-          <div
-            className={cx(className, "relative bg-white rounded")}
-            role="dialog"
-          >
+          <div className={cx(className, "relative bg-white rounded")}>
```

Comment:
`role=dialog` is now set just on the ModalContent component, not any of its parents.